templates/implnet_ops_SOURCEVAL.py

```diff
@@ -474,62 +409,26 @@ def gleanerio(context, mode, source):
     validate_docker_image(IMAGE)
 
     try:
-        # setup data/body for  container create
-        data = {}
-        data["Image"] = IMAGE
-        data["WorkingDir"] = WorkingDir
-        # data["Entrypoint"] = Entrypoint
-        data["Cmd"] = ARGS
-
-        environment_variables = [
-            f"MINIO_ADDRESS={GLEANER_MINIO_ADDRESS}",
-            f"MINIO_PORT={GLEANER_MINIO_PORT}",
-            f"MINIO_USE_SSL={GLEANER_MINIO_USE_SSL}",
-            f"MINIO_SECRET_KEY={GLEANER_MINIO_SECRET_KEY}",
-            f"MINIO_ACCESS_KEY={GLEANER_MINIO_ACCESS_KEY}",
-            f"MINIO_BUCKET={GLEANER_MINIO_BUCKET}",
-            f"SPARQL_ENDPOINT={_graphEndpoint()}",
-            f"GLEANER_HEADLESS_ENDPOINT={GLEANER_HEADLESS_ENDPOINT}",
-            f"GLEANER_HEADLESS_NETWORK={GLEANER_HEADLESS_NETWORK}",
```

Comment:
some of the env stuff for dagster is still a bit opaque but it seems that this is unnecessary and it runs fine without it as long as we run `container_context = run_container_context.merge(op_container_context)`